cmd/fpga_crihook/main.go

```diff
@@ -124,12 +128,71 @@ func (bitStream *opaeBitStream) validate() error {
 	return nil
 }
 
+func getFpgaConfArgs(dev string) ([]string, error) {
+	realDevPath, err := findSysFsDevice(dev)
+	if err != nil {
+		return nil, errors.Wrapf(err, "can't detect sysfs entry for %s", dev)
```

Comment:
And here there's no need to wrap the error, because it originates from our function where errors are augmented with stack traces already.